### PR TITLE
fix(SITES-2854): reduce the margin-right size of icon

### PR DESCRIPTION
**Pull request by RazvanPRA**
https://jira.corp.adobe.com/browse/SITES-2854

### CODE REVIEW

**Comment by RazvanPRA** [comment]
@sandru85 I can merge this PR? This is the fix for SITES-2854 on open source coral-spectrum

**Comment by sandru85** [comment]
@Pareesh can you please review?
@RazvanPRA let's wait for a review from the dedicated team

**Review by Pareesh — APPROVED** [review]

**Pull request merged**
